src/winmain.c

```diff
@@ -3677,6 +3677,14 @@ exit_mintty(void)
 {
   report_pos();
 
+  // bring next window to top
+  if (sync_level()) {
```

Comment:
I'm not sure whether this condition is okay.
Should this be `>= 2` or `>= 3`? Or, should `cfg.tabbar` be also checked?